src/traces/ohlc/attributes.js

```diff
@@ -102,7 +102,7 @@ module.exports = {
         valType: 'number',
         min: 0,
         max: 0.5,
-        dflt: 0.05,
+        dflt: 0.1,
```

Comment:
done in https://github.com/plotly/plotly.js/pull/1020/commits/b98835c4e9b99f125bd4ff7b144d8680918b2cff